src/main/java/frc/robot/subsystems/climber/commands/AutoClimb.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.climber.Climber;
+
+public class AutoClimb extends SequentialCommandGroup {
+
+    private final Climber climber;
+
+    public AutoClimb(Climber climber) {

```

Comment:
1) You have both `Climb` and `AutoClimb`. The commands' names don't explain their differences.
2) This command name contains the word "climb", but there is no actual climbing that runs in it. It only shifts the gear and disengages the stopper.